ssg/oval_object_model/general.py

```diff
@@ -0,0 +1,169 @@
+from ..constants import oval_namespace as oval_ns
+from ..xml import ElementTree
+
+# ----- Constants
+
+
+STR_TO_BOOL = {
+    "false": False,
+    "False": False,
+    "true": True,
+    "True": True,
+}
+
+BOOL_TO_STR = {True: "true", False: "false"}
+
+OVAL_NAMESPACES = {
+    "oval": "http://oval.mitre.org/XMLSchema/oval-common-5",
```

Comment:
If this isn't already defined in `ssg/constants.py` you should put it there and not here.

Reply:
Should I move all constants from the OVAL object model to the `ssg/constants.py` file?

Reply:
In general, I would say that constants that have a likelihood of being used elsewhere in the project should be moved to ssg/constants.py. Namespace URLs are an example of constants that should definitely go to ssg/constants.py. On the other hand, you can still keep some constants in this module, but those should be constants that are extremely unique for this module and you don't expect reusing them anywhere else in future.